doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,81 @@
+.. _box_introspection-box_stat_memtx:
+
+box.stat.memtx()
+================
+
+.. module:: box.stat
+
+.. function:: memtx()
+
+    Shows memtx-storage-engine activity.
+
+.. _box_introspection-box_stat_memtx_tx:
+
+box.stat.memtx().tx
+-------------------
+
+* ``tx`` shows the statistics of the memtx transactional manager,
+  which is responsible for transactions (``box.stat.memtx().tx.txn``)
+  and multiversion concurrency control (``box.stat.memtx().tx.mvcc``).
+
+  * ``box.stat.memtx().tx.txn`` shows memory allocation related to transactions.
+
+    It consists of the following sections:
+
+    * ``statements`` are *transaction statements*.
+      As an example, consider a user starting a transaction with
+      ``space:replace{0, 1}`` within this transaction. Under the hood,
+      this operation becomes a statement for this transaction.
+    * ``user`` is the memory that a user allocated within
+      the current transaction using the Tarantool C API function
+      :ref:`box_txn_alloc() <txn-box_txn_alloc>`.
+    * ``system`` is the memory allocated for internal needs
+      (for example, logs) and savepoints.
+
+      For each section, Tarantool reports the following statistics:
+
+      * ``total`` is the number of bytes that are currently allocated in memtx
+        for all transactions within the section scope.
+      * ``avg`` is the average number of bytes that a single transaction uses
+        (equals ``total`` / number of open transactions).
+      * ``max`` is the maximal number of bytes that a single transaction uses.
+
+  * ``box.stat.memtx().tx.mvcc`` shows memory allocation related to multiversion
+    concurrency control (MVCC). MVCC is reponsible for isolating transactions.
+    It reveals conflicts and makes sure that tuples that do not belong to a particular
+    space but were (or could be) read by some transaction were not deleted.
+
+    It consists of the following sections:
+
+    * ``trackers`` is the memory allocated for *trackers* of transaction reads.
+      Like in the previous sections, Tarantool reports the total, average
```

Comment:
Nit: The introductory clause `Like in the previous sections` sounds a bit awkward to me. As a reference reader, I would just search (ctrl+F) for a specific name, not read the whole page. Thus, `previous sections` without a clickable link forces me to start reading the content I didn't need)

Maybe, create a subsection with `total`, `avg`, and `max` definitions and link it from all applicable objects?